ch04/4-1/samplecli-builder/Cargo.toml

```
@@ -1,10 +1,10 @@
 [package]
 name = "samplecli-builder"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["PACKAGE AUTHOR <package_author@example.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = "3.0.0-beta.1"
+clap = "3.0.0-beta.2"
```

これだとbeta.3が出たときにまた壊れてしまう可能性があるので以下のようにしておくのがいいかもしれません。

```suggestion
clap = "=3.0.0-beta.2"
```

---

たしかに、Cargo.lockがない状態（手元で写経）のことを考えると`=`を入れておいた方がよさそうですね。

---

@dalance @matsu7874 修正しましたのでご確認をお願いします。

---

ありがとうございます。問題ないと思います。